features/src/main/feature/camel-features.xml

```diff
@@ -2004,15 +1998,15 @@
     </feature>
     <feature name='camel-redis' version='${project.version}' start-level='50'>
         <feature version='${camel.osgi.version.range}'>camel-core</feature>
-        <feature version='[2.17,3)'>jackson</feature>
+        <feature version='[2.17.1,3)'>jackson</feature>
```

Comment:
Why not `[2.17,2.18)` like the rest?

Reply:
It was already with 3 in max so I don't change it but I can align. May be it will be better to add the range as a property?

Reply:
As you wish, you can set `[2.17,2.18)` directly, both approaches are fine with me